modules/olm-operatorgroups-target-namespace.adoc

```diff
@@ -5,8 +5,8 @@
 [id="olm-operatorgroups-target-namespace_{context}"]
 = Target namespace selection
 
-The set of namespaces for the OperatorGroup is specified using a label selector
-with the `spec.selector` field:
+You can explicitly name the target namespace for an OperatorGroup using the
+`spec.targetNamespaces` field:
```

Comment:
@adellape, I thought we had been using _field_ for GUI fields, and _parameter_ for YAML?